Topics/User_Experience/10_Heuristics.md

```diff
@@ -0,0 +1,65 @@
+# "Topic: 10 Heuristics of User Interface Design by Jakob Nielsen
+
+## table of contents, the 10 heuristic rules:
```

Comment:
There is no table of contents here. Maybe change this line to "Introduction"

Reply:
Solved, thanks!